reference/docs-conceptual/learn/deep-dives/visualize-parameter-binding.md

```diff
@@ -0,0 +1,252 @@
+---
+description: This article demonstrates how to use the Trace-Command cmdlet to visualize the parameter binding process in PowerShell.
+ms.date: 05/17/2024
+ms.topic: how-to
+title: Visualize parameter binding
+---
+# Visualize parameter binding
+
+Parameter binding is the process that PowerShell uses to determine which parameter set is being used
+and to associate (bind) values to the parameters of a command. These values can come from the
+command line and the pipeline.
+
+The parameter binding process starts by binding named and positional command-line arguments. After
+binding command-line arguments, PowerShell tries to bind any pipeline input. There are two ways that
+values are bound from the pipeline. Parameters that accept pipeline input have one or both of the
+following attributes:
+
+- [ValueFromPipeline][01] - The value from the pipeline is bound to the parameter based on its type.
+  The type of the argument must match the type of the parameter.
+- [ValueFromPipelineByPropertyName][02] - The value from the pipeline is bound to the parameter
+  based on its name. The object in the pipeline must have a property that matches the name of the
+  parameter or one of its aliases. The type of the property must match or be convertible to the type
+  of the parameter.
+
+For more information about parameter binding, see [about_Parameter_Binding][03].
+
+## Use `Trace-Command` to visualize parameter binding
```

Comment:
Are we switching to using markup in headers? I'm okay with this personally, I prefer it - but I thought our style guide was to avoid them.

```suggestion
## Use Trace-Command to visualize parameter binding
```

Reply:
Yes, on a limited basis.